build.sbt

```diff
@@ -596,10 +595,16 @@ lazy val componentModulesPaths =
   )
   val thirdPartyModFiles = thirdPartyMods.map(_.data)
   val arrow              = (`runtime-language-arrow` / Compile / packageBin).value
+  val syntax             = (`syntax-rust-definition` / Compile / packageBin).value
+  val ydoc               = (`ydoc-server` / Compile / packageBin).value
+  val profilingUtils     = (`profiling-utils` / Compile / packageBin).value
   val runtime            = (`runtime-fat-jar` / assembly / assemblyOutputPath).value
   val ourMods = Seq(
     runtime,
-    arrow
+    arrow,
```

Comment:
Shouldn't we compute `outMods` by scanning projects that contain `javaModuleName`?

Reply:
This is not a bad idea. But it won't save us a lot of manual work, since we will still need to explicitly get modules inside module-path in various other places, like `runtime-integration-tests`, `language-server/test` or `project-manager/test`, and others.